python/pyspark/ml/feature.py

```diff
@@ -2674,6 +2674,18 @@ def findSynonyms(self, word, num):
             word = _convert_to_vector(word)
         return self._call_java("findSynonyms", word, num)
 
+    @since("2.2.0")
+    def findSynonymsArray(self, wordVector, num):
+        """
+        Find "num" number of words closest in similarity to "word".
+        word can be a string or vector representation.
+        Returns a dataframe with two fields word and similarity (which
+        gives the cosine similarity).
+        """
+        # if not isinstance(wordVector, basestring):
```

Comment:
Why is this commented out? There are two versions of `findSynonymsArray` on the Scala side just like there are for `findSynonyms` - we need it to work for both a string and vector input.